relay-common/src/time.rs

```diff
@@ -139,13 +144,20 @@ impl std::ops::Sub for UnixTimestamp {
     }
 }
 
+#[derive(Debug)]
 /// An error returned from parsing [`UnixTimestamp`].
 pub struct ParseUnixTimestampError(());
 
 impl std::str::FromStr for UnixTimestamp {
     type Err = ParseUnixTimestampError;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if let Ok(datetime) = s.parse::<DateTime<Utc>>() {
```

Comment:
The `UnixTimestamp` type was quite odd in that the string deserialization supports iso timestamps, but the `FromStr` implementation did not. I changed this now to allow either.